_rules/object-has-accessible-name-8fc3b6.md

```diff
@@ -50,11 +50,7 @@ The [MIME type][] of the resource embedded in the `data` attribute impacts how t
 
 Testing that the [accessible name][] describes the purpose of the `object` element is not part of this rule and must be tested separately.
 
-Non-supported media formats make screen readers render the text content of the element instead of other attributes.
-
-`Object` elements without an accessible name are ignored by assistive technologies unless they have an [explicit role][].
-
-When the object resource is not loaded, the fallback content is rendered as shown in the Inapplicable Example: "This `object` element does not need an accessible name because it loads no image, audio, or video."
+When the object resource is not loaded, the fallback content is rendered as shown in the Inapplicable Example: "This `object` element does not need an accessible name because it loads no image, audio, or video.". When screen readers encounter an unsupported media format they will also use the fallback content instead of other attributes.
```

Comment:
```suggestion
When the object resource is not loaded, the fallback content, if present, is rendered as shown in the Inapplicable Example: "This `object` element does not need an accessible name because it loads no image, audio, or video.". When screen readers encounter an unsupported media format they will also use the fallback content instead of other attributes.
```